swerve100/src/main/java/frc/robot/subsystems/SwerveDriveSubsystem.java

```diff
@@ -223,7 +223,7 @@ public SwerveDriveSubsystem(double currentLimit) throws IOException {
                 kMaxSpeedMetersPerSecond = 5;
                 kMaxAccelerationMetersPerSecondSquared = 10;
                 kMaxAngularSpeedRadiansPerSecond = 5;
-                kMaxAngularSpeedRadiansPerSecondSquared = 5;
+                kMaxAngularSpeedRadiansPerSecondSquared = 10;
```

Comment:
i think you put this in the wrong place, the roborio called FROM_8048 is owned by team 8048, my "other" team.  (I ran our code on their robot briefly to see if i could debug a problem they were having with their andymark swerve modules.)